agent/Agent.js

```diff
@@ -373,6 +378,18 @@ class Agent extends EventEmitter {
     this.emit('unmount', id);
     this.ids.delete(component);
   }
+
+  _onScroll() {
+    if (!this._scrollUpdate) {
+      this._scrollUpdate = true;
+      window.requestAnimationFrame(this._updateScroll.bind(this));
```

Comment:
this will generate a new function at each call, better to store the *bound* function as a property (eg. `this._updateScroll = this._updateScroll.bind(this);` inside the constructor)